rust/sysdb/src/sqlite.rs

```diff
@@ -348,7 +348,23 @@ impl SqliteSysDb {
         name: Option<String>,
         metadata: Option<CollectionMetadataUpdate>,
         dimension: Option<u32>,
+        configuration: Option<UpdateCollectionConfiguration>,
     ) -> Result<(), UpdateCollectionError> {
+        let mut configuration_json_str = None;
+        if let Some(configuration) = configuration {
+            let collections = self
+                .get_collections(Some(collection_id), None, None, None, None, 0)
```

Comment:
this should be part of the transaction, otherwise there's a possible race condition

Reply:
this is still not part of the transaction